jgrapht-core/src/main/java/org/jgrapht/alg/GabowStrongConnectivityInspector.java

```diff
@@ -0,0 +1,260 @@
+package org.jgrapht.alg;
```

Comment:
All files need the standard JGraphT header (including both the project-level header and the file-level header).